cdap-ui/app/cdap/components/PipelineScheduler/Store/index.js

```diff
@@ -215,7 +217,18 @@ const schedule = (state = DEFAULT_SCHEDULE_OPTIONS, action = defaultAction) => {
       };
     case ACTIONS.SET_CURRENT_BACKEND_SCHEDULE: {
       let {currentBackendSchedule} = action.payload;
+      const getProfileCustomizations = (scheduleProperties = {}) => {
```

Comment:
Can we extract this function out somewhere? I see this extraction of customizations from properties happening in multiple places